src/animation/AnimationUtils.js

```diff
@@ -158,6 +158,50 @@ var AnimationUtils = {
 
 		}
 
+	},
+
+	clone: function ( source ) {
+
+		var sourceNodes = {};
+		var clonedNodes = {};
+
+		var clone = source.clone();
+
+		source.traverse( function ( node ) {
+
+			sourceNodes[ node.name ] = node;
+
+		} );
+
+		clone.traverse( function ( node ) {
+
+			clonedNodes[ node.name ] = node;
+
+		} );
+
+		clone.traverse( function ( node ) {
+
+			if ( ! node.isSkinnedMesh ) return;
+
+			var clonedMesh = node;
+			var sourceMesh = sourceNodes[ node.name ];
+			var sourceBones = sourceMesh.skeleton.bones;
+
+			clonedMesh.skeleton = sourceMesh.skeleton.clone();
+			clonedMesh.bindMatrix = sourceMesh.bindMatrix;
```

Comment:
We should copy the matrix, rather than we share?

Reply:
Done ✅